guide/payments/coin-selection.md

```diff
@@ -0,0 +1,157 @@
+
+- 0.25 BTC
+- 0.01 BTC
+- 0.56 BTC
+- 0.04 BTC
+- 0.24 BTC
+- ... etc
+
+These fractions are all UTXOs (unspent change) from previously mined coins.
+
+Coin selection, therefore, is the choosing of which UTXOs to fund a bitcoin transaction with, in order words, the transactions inputs. 
+
+## Design Problem
+
+Because each UTXO can be traced back on a public ledger to it's newly minted state (a whole bitcoin), we can essentially trace the digital footprints of payments, exposing private data of senders, receivers, and their various wallet balances along the way. Coin selection, whether an automated or manual process, will most likely result in the breach of privacy for both you as well as your previous senders of bitcoin. However, there are various ways of mitigating how much data is exposed during each transaction. We can reduce this security risk in a few different ways, however, we are left with a design challenge.
+
+How do we account for coin selection within a user flow of creating a bitcoin transaction, whilst minimising the risk of a privacy breach? 
+
+### Scenario
+
+`As a user I want to make a 0.25 btc payment to my friend Patrick and reveal the least amount of information about the bitcoin I own, and I want it to be cheap. It is not an urgent payment.`
+
+My wallet currently has a balance of **1.1726 BTC**, and is comprised of 5 UTXOs:
+
+- 0.4363 from Ed
+- 0.4363 from Jelena
+- 0.1 from Ed
+- 0.1 from Patrick
+- 0.1 from Bill
+
+These UTXOs can also be grouped into "clusters" from the individual senders (see diagram). 
+
+Ideally, if I were to send a payment of 0.25 BTC to Patrick, I would have a cluster of UTXOs that have previously come from one of his own incoming transactions. Therefore, I do not reveal any additional information about the UTXOs I control to the payee. However, unfortunately I do not have a cluster only known to Patrick with enough UTXOs totalling the amount I wish to send. Therefore, I am going to have to select coins (UTXOs) to fund my transaction that have not originated from one of Patrick's addresses, thereby partially exposing coins from that cluster, as well as that of another (Ed, Jelena, or Bill). Ideally, we want to expose the least amount of information possible to Patrick. 
+
+![](https://i.imgur.com/LtTUq5u.png)
+
+### The Transaction
+
+To complete Patrick's payment request of 0.25 BTC, I am going to have to select which UTXOs to fund the transaction with. 
+
+Because the payment is to Patrick, it makes sense to use the 0.1 BTC (UTXO) from his address, as this does not comprimise anyone else's privacy. However, I still have 0.15 BTC remaining in order to fund the transaction. 
+
+This transaction could be funded in multiple different ways:
+
+- **Option 1:**
```

Comment:
I am unsure if it's just me, but imo this is all better explained visually. There are just way too many numbers and names for a person to understand what's going on here.